pegasus/sites.v3/hourofcode.com/public/events/2016/index.haml

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
Is this file a copy-and-paste of the previous years, substituting the year in `HocSignup2016`? If so, any chance this could be rewritten to share the common code?